lib/iris/aux_factory.py

```diff
@@ -817,30 +822,21 @@ def make_coord(self, coord_dims_func):
 
         # Build the points array.
         nd_points_by_key = self._remap(dependency_dims, derived_dims)
-        points_shape = self._shape(nd_points_by_key)
-
-        # Calculate the nsigma slice.
-        nsigma_slice = [slice(None)] * len(derived_dims)
-        dim, = dependency_dims['zlev']
-        index = derived_dims.index(dim)
-        nsigma_slice[index] = slice(0, int(nd_points_by_key['nsigma']))
-        nsigma_slice = tuple(nsigma_slice)
 
+        nsigma, = nd_points_by_key['nsigma']
```

Comment:
@pp-mo Subtle unpacking ... `[nsigma] = nd_points_by_key['nsigma']` is an alternative, less subtle pattern ... you're choice, it's a minor point.